pkg/rancher-desktop/pages/Containers.vue

```diff
@@ -331,25 +331,27 @@ export default {
       return { content: sha };
     },
     getUniquePorts(ports) {
```

Comment:
I was going to complain that `ports` isn't typed, then realized… I don't think this is TS at all >_<